### PR TITLE
Added buggy Python and C code

### PR DESCRIPTION
**Pull request by themohitnair**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-# Galti-Se-Mistake
-A repository used for our "Good First Bug" Sessions where we walk the participants through the process of contributions on a Git Version control environment
+A repository used for our "Good First Bug" Sessions where we walk the participants through the process of contributions on a Git Version control environment 
+Enter your full name and github username in the format <name> ==> <username> under 'CONTRIBUTORS' if you have contributed to the code in this repository.
+
+CONTRIBUTORS
+Mukti ==> MuktiCommunity
```
Comment (MuktiCommunity) [inline README.md:5]: Remove it not needed

File: chaatr.c
Comment (MuktiCommunity) [inline chaatr.c:1]: Add C and Python in separate files for each language and also add level of difficulty in the FileName

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (themohitnair) [reply README.md:5]: Won't it be good if they could contribute to README as well? I don't think they'll be able to add anything to the README if that is removed.

File: chaatr.c
Comment (themohitnair) [reply chaatr.c:1]: Didn't get you.

File: README.md (hunk 1 above)
Comment (MuktiCommunity) [reply README.md:5]: I meant, Dont include Mukti ===> MuktiCommnity
Add both of our names as an example
~Het

File: chaatr.c
Comment (MuktiCommunity) [reply chaatr.c:1]: Make seperate Folders I meant.
/Python which will have the Pyhton codes in them and each filename will include their project name and their difficulty.
eg> /Python/calculator_Intermediate.py

File: README.md (hunk 1 above)
Comment (themohitnair) [reply README.md:5]: Okay cool, got it.

File: chaatr.c
Comment (themohitnair) [reply chaatr.c:1]: cool, thanks, will make changes right away.

**Pull request closed without merging**